eclim-debug.el

```diff
@@ -91,11 +91,19 @@
       (add-hook 'comint-output-filter-functions
                 (lambda (txt) (eclim--debug-attach-when-ready txt project port))))))
 
+(defun eclim-debug/jdb (command)
+  (let ((buffer (current-buffer)))
+    (toggle-maximize-buffer)
+    (switch-to-buffer-other-window buffer t)
+    (jdb command)
+    (switch-to-buffer-other-window buffer t)))
```

Comment:
Is there a better way to do this? I bet there is. I'd be happy if anyone would enlighten me